### ISSUE TITLE
Make package including custom map

### ISSUE THREAD

**Issue by Ahnhojin1223** [carla-simulator/carla#7897]
CARLA version: 0.9.15
Platform/OS: Ubuntu.20.04

If I create a custom map using Roadrunner, put it in the Imoprt folder, type "make import ARGS="--package=map_name" into command, that custom map is created in Carla. After that, type "make package ARGS="--package=map_name" into command, only the package in that map is created.
By the way, if I use "make package" to create the entire package including the custom map, the generated package does not contain the custom map. Is there a way to fix this???

**Comment by PatrickPromitzer** [comment]
Hi,
You need to add the custom Map in the packaging settings.
![image](https://github.com/carla-simulator/carla/assets/152880896/44fa695d-1e82-4543-83be-834ce9c10fea)

With using "make package", Carla is creating a package with the Maps listed in the packaging settings.

**Comment by Madecu** [comment]
Hi @Ahnhojin1223,

You can also do a package of your map as an additional map. You can edit /Content/Carla/Config/AdditionalMaps.Package.json and add your map and then run _make package ARGS="--package=AdditionalMaps --clean-intermediate"_

**Comment by chuanye-Wang** [comment]
@Madecu The same issue, is there any way to export my custom map individually as an umap, so I can import it to another computer? Thanks

**Comment by Madecu** [comment]
Hi @chuanye-Wang,

You can locate the .umap file in the content folder on your disk. However, ensure that all the assets referenced in the map are in the same locations; otherwise, the references will be broken.

**Comment by chuanye-Wang** [comment]
Thank you, I solved this issue yesterday, but still have some question out of curious. I followed the guide to create my cusMap.Package.json like this, and it worked.
'''JSON
{
  "maps": [
    {
        "path": "/Game/map_package/Maps/cusMap",
        "name": "cusMap",
        "use_carla_materials": true
      }
  ],
  "props": []
}
'''
But I took a look at Town06_Opt.Package.json, and it was quite complicated:
![img_v3_02d9_1b1525fc-c9d0-47ca-9516-e96e0347c2ag](https://github.com/user-attachments/assets/62001528-d20a-425a-8295-4112aaa0a27d)

My cusMap worked fine, so I just ignored the difference between them yesterday.
